app/models/notebook.rb

```diff
@@ -20,6 +20,7 @@ class Notebook < ActiveRecord::Base
   has_many :revisions, dependent: :destroy
   has_many :reviews, dependent: :destroy
   has_many :subscriptions, as: :sub, dependent: :destroy
+  has_one :deprecated_notebbook
```

Comment:
To add the ability to search boost on this, in this block - https://github.com/nbgallery/nbgallery/blob/5ed6b7864c9349ef57e4b35a6b98b34ab441f3e4/app/models/notebook.rb#L35 down near packages, add another field called active or something that is a boolean and set to true if the notebook is NOT active (Easier to positive boost a good value since you can't negatively boost a bad value).  Then we need to toy with the weight further down in the file but we at least need it set.  After this is added, a Notebook.reindex will add it to solr